src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -163,7 +163,7 @@
     'SCCF_chan_wavelen': ('sensorCentralFrequency', 'double'),
     'QI_with_FC': ('percentConfidenceWithForecast', 'float'),
     'QI_without_FC': ('percentConfidenceWithoutForecast', 'float'),
-    'Data_Vertical_Velocity': ('windUpward', 'float'),
+    'Data_Vertical_Velocity': ('instantaneousAltitudeRate', 'float'),
```

Comment:
excellent thank you for the tie to 
https://github.com/JCSDA-internal/ioda/blob/eaacb84f813add08fce4cf22530a99d48895ec76/share/ioda/yaml/validation/ObsSpace.yaml#L1079